lib/helpers/indexes/isIndexEqual.js

```diff
@@ -28,15 +28,19 @@ module.exports = function isIndexEqual(key, options, dbIndex) {
   //   textIndexVersion: 3
```

Comment:
A compound index with text and non-text fields will look like this:
  `{ name: 'text', age:1 }` becomes:
```js
   {
     v: 2,
     key: { _fts: 'text', _ftsx: 1, age: 1 },
     name: 'name_text_age_1',
     ns: 'test.tests',
     background: true,
     weights: { name: 1 },
     default_language: 'english',
     language_override: 'language',
     textIndexVersion: 3
   }
```